server/src/domain/media/media.util.ts

```diff
@@ -555,3 +562,58 @@ export class VAAPIConfig extends BaseHWConfig {
     return this.config.cqMode !== CQMode.ICQ || this.config.targetVideoCodec === VideoCodec.VP9;
   }
 }
+
+export class RKMPPConfig extends BaseHWConfig {
+  getOptions(videoStream: VideoStreamInfo, audioStream?: AudioStreamInfo | undefined): TranscodeOptions {
+    const options = super.getOptions(videoStream, audioStream);
+    options.ffmpegPath = 'ffmpeg_mpp';
```

Comment:
why the need for a different path? Can you instead re-link  `/usr/bin/ffmpeg` via a volume mount instead or something like that?

Reply:
it allows to use the software transcoding fallback using the normal ffmpeg binary from the docker image.
the patched rockship ffmpeg uses hardware even if you just specify `-c:v hevc` without asking for hardware transcoding :-)
also it might have bugs in the hardware decoder for some videos; thus it needs the fallback to the properly working ffmpeg binary

Reply:
Ah, I see. Does the user have to manually do anything to make this binary available?

Reply:
install ffmpeg for the rockchip SBC from a specific ubuntu ppa (mentioned in `hwaccel-rkmpp.yml`)